Packages/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "dependencies": {
```

Comment:
What's this file doing here?

Reply:
it appeared out of the blue so I left it there...

Reply:
That file is usually generated inside unity projects but here it's in the root folder and not `/NitroxUnity`. So it's save to delete.